src/control/cmd/dmg/check.go

```diff
@@ -148,3 +184,60 @@ func (cmd *checkPropCmd) Execute(_ []string) error {
 
 	return nil
 }
+
+type repairSeqNum uint64
+
+func (r repairSeqNum) String() string {
+	return fmt.Sprintf("0x%x", uint64(r))
+}
+
+func (r *repairSeqNum) UnmarshalFlag(value string) error {
+	var val uint64
+	var err error
+	if strings.HasPrefix(value, "0x") {
+		cleaned := strings.Replace(value, "0x", "", -1)
+		val, err = strconv.ParseUint(cleaned, 16, 64)
+	} else {
+		val, err = strconv.ParseUint(value, 10, 64)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	*r = repairSeqNum(val)
+	return nil
+}
+
+type checkRepairCmd struct {
+	cmdutil.LogCmd
+	cfgCmd
+	ctlInvokerCmd
+	jsonOutputCmd
+
+	ForAll bool `short:"f" long:"for-all" description:"Take the same action for all inconsistencies with the same class."`
+
+	Args struct {
+		SeqNum repairSeqNum `positional-arg-name:"[seq-num]" required:"1"`
+		Action int32        `positional-arg-name:"[action]" required:"1"`
```

Comment:
What are the valid values here? Are they presented as output from another checker command? Same question for sequence numbers.